src/components/HeartButton.js

```diff
@@ -0,0 +1,13 @@
+import { ReactComponent as HeartIcon } from "assets/images/ic_heart.svg";
+import styles from "./HeartButton.module.css";
+
+const HeartButton = ({ favoriteCount }) => {
+  return (
+    <button aria-label="좋아요 버튼" className={styles.heartButton}>
```

Comment:
P2:
button 태그가 어떤 맥락에서 쓰일지 모르니 가능하면 type을 명시해주세요.

```suggestion
    <button aria-label="좋아요 버튼" className={styles.heartButton} type='button'>
```